src/styles/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url('../images/star-active.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+    display: flex;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+  }
+
+  &__star:nth-child(5) {
```

Comment:
Use nesting:

Just like you used & to concat class names, you should use it for pseudo-selectors also:
```
.stars {
  ...
  &__star {
    ...
    &:nth-child(5) {
      background-image: url(...)
    }
  }
}
```